annotate.py

```diff
@@ -47,6 +61,8 @@ def draw_circle(event,x,y,flags,param):
         ## click on image to grab pixel coordinates
         ## s to close image and exit program
         ## r to write coordinates to file (you will be prompted for the PMT feature ID)
+        ## f to select new PMT
```

Comment:
update with all keystroke features (or just remove to only need to maintain the README)